### PR TITLE
Use a dedicated, tainted Heapster node in Kubemark5000 tests 

### PR DESCRIPTION
**Pull request by oxddr**
See https://github.com/kubernetes/kubernetes/pull/85797, https://github.com/kubernetes/perf-tests/pull/913 and https://github.com/kubernetes/kubernetes/issues/85683. 

/assign @wojtek-t

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml [hunk 1]
```diff
@@ -81,6 +81,11 @@ presets:
     value: "false"
   - name: HOLLOW_PROXY_TEST_ARGS
     value: "--use-real-proxier=false"
+  - name: HEAPSTER_KUBELET_TEST_ARGS
+    value: "--register-with-taints=monitoring=:PreferNoSchedule"
+  - name: HEAPSTER_MACHINE_TYPE
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml]: Won't it create heapster node also in the base cluster?

Comment (oxddr) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml]: This change should add an extra, tainted node to the base cluster.

Comment (wojtek-t) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml]: OK - that was a stupid question.

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml [hunk 2]
```diff
@@ -81,6 +81,11 @@ presets:
     value: "false"
   - name: HOLLOW_PROXY_TEST_ARGS
     value: "--use-real-proxier=false"
+  - name: HEAPSTER_KUBELET_TEST_ARGS
+    value: "--register-with-taints=monitoring=:PreferNoSchedule"
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml]: Can we switch to `NoSchedule` - with `PreferNoSchedule` we have exactly 0 guarantee for anything, and i don't want to end-up in a situation that 2 weeks later we will be debugging exactly the same problem.

Comment (oxddr) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml]: Done.

Comment (wojtek-t) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml]: Not done

**Comment by oxddr** [comment]
PTAL.

**Pull request merged**